opentelekomcloud/services/ecs/resource_opentelekomcloud_ecs_instance_v1.go

```diff
@@ -126,6 +126,12 @@ func ResourceEcsInstanceV1() *schema.Resource {
 				ForceNew: true,
 				Computed: true,
 			},
+			"system_disk_kms_id": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				ForceNew:    true,
+				DefaultFunc: schema.EnvDefaultFunc("OS_KMS_ID", nil),
```

Comment:
Not sure, if system disk should be encrypted by default when `OS_KMS_ID` is set

Reply:
we use same `DefaultFunc` in `data_disk.*.kms_id` field

Reply:
But this will still change existing behaviour when system disk is not encrypted in the same case